File: src/main/java/bxx2004/bump/util/GuiItems.java
package bxx2004.bump.util;

import bxx2004.bump.Bump;
import io.github.thebusybiscuit.slimefun4.libraries.dough.items.CustomItemStack;
import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;

public final class GuiItems {
    private GuiItems() {}

    public static ItemStack APPRAISE_BUTTON = new CustomItemStack(
        Material.NAME_TAG,
        Bump.getLocalization().getString("gui.appraise.name"),
        Bump.getLocalization().getStringArray("gui.appraise.lore")
    );

    public static ItemStack APPRAISE_PAPER = new CustomItemStack(
        Material.PAPER,
        Bump.getLocalization().getString("gui.appraisal_paper.name"),
        Bump.getLocalization().getStringArray("gui.appraisal_paper.lore")
    );
}

File: src/main/kotlin/io/github/slimefunguguproject/bump/utils/items/GuiItems.kt
package io.github.slimefunguguproject.bump.utils.items

import io.github.slimefunguguproject.bump.Bump
import io.github.slimefunguguproject.bump.api.appraise.AppraiseType
import io.github.thebusybiscuit.slimefun4.libraries.dough.items.CustomItemStack
import org.bukkit.Material
import org.bukkit.inventory.ItemStack

// TODO: Remove this, gui items are defined in the menus directly
object GuiItems {
    val APPRAISE_BUTTON: ItemStack = CustomItemStack(
        Material.NAME_TAG,
        Bump.localization.getString("gui.appraise.name"),
        *Bump.localization.getStringArray("gui.appraise.lore")
    )
    val APPRAISE_PAPER: ItemStack = CustomItemStack(
        Material.PAPER,
        Bump.localization.getString("gui.appraisal_paper.name"),
        *Bump.localization.getStringArray("gui.appraisal_paper.lore")
    )
    val GRIND_BUTTON: ItemStack = CustomItemStack(
        Material.GRINDSTONE,
        Bump.localization.getString("gui.grind.name"),
        *Bump.localization.getStringArray("gui.grind.lore")
    )

    fun appraiseTypeSelector(type: AppraiseType): ItemStack {
        val lore = AppraiseUtils.getDescriptionLore(type) +
            Bump.localization.getStringList("gui.appraise_type_selector.lore")

        return CustomItemStack(
            Material.MAP,
            Bump.localization.getString("gui.appraise_type_selector.name", type.name),
            lore
        )
    }
}
